DependencyInjection/Configuration.php

```diff
@@ -46,6 +46,19 @@ public function getConfigTreeBuilder()
                 ->defaultValue('sonata.dashboard.block.container')
                 ->cannotBeEmpty()
             ->end()
+            ->arrayNode('templates')
+                ->addDefaultsIfNotSet()
+                ->children()
+                    ->scalarNode('compose')
+                        ->defaultValue('SonataDashboardBundle:DashboardAdmin:compose.html.twig')
+                        ->info('This value sets the composer template.')
```

Comment:
it corresponds to the "composer" having two templates ("compose" and "compose_container_show") same as for SonataPageBundle